man/man1/mail.1
.\" @(#)mail.1 1.1 92/07/30 SMI; from UCB 4.2 and S5
.ds ~ \u\(ap\d
.TH MAIL 1 "19 December 1988"
.SH NAME
mail, Mail \- read or send mail messages
.SH SYNOPSIS
.B Mail
.RB [ " \-deHinNUv " ]
.RB [ " \-f " 
.RI [ " filename " | " \fB+\fPfolder"
] ]
.RB [ " \-T "
.IR file " ]"
.if n .ti +0.5i
.RB [ " \-u "
.IR user " ]"
.LP
.B Mail
.RB [ " \-dFinUv " ]
.RB [ " \-h "
.IR number " ]"
.RB [ " \-r "
.IR address " ]"
.RB [ " \-s "
.IR subject " ]"
.if n .ti +0.5i
.I recipient
\&.\|.\|.
.LP
.B /usr/ucb/mail
\&.\|.\|.
.SH DESCRIPTION
.IX  "mail command"  ""  "\fLmail\fP \(em send and receive mail"  ""  PAGE START
.IX  "send and receive mail"  ""  "send and receive mail \(em \fLmail\fP"  ""  PAGE START
.IX  "read mail"  ""  "read mail \(em \fLmail\fP"  ""  PAGE START
.IX  communications  mail  ""  "\fLmail\fP \(em send and receive mail"  PAGE START
.LP
.B mail
is a comfortable, flexible, interactive program for composing, sending
and receiving electronic messages.  While reading messages,
.B mail
provides you with
commands to browse, display, save, delete, and respond to messages.
While sending mail, 
.B mail
allows editing and reviewing of messages being composed, and the
inclusion of text from files or other messages.
.LP
Incoming mail is stored in the
.I system mailbox 
for each user.  This is a file named after the user in
.BR /var/spool/mail .
.B mail
normally looks in this file for incoming messages, but you
can use the
.SB MAIL
environment variable to have it look in a different file.
When you read a message, it is
marked to be moved to a secondary file for storage. 
This secondary file, called the 
.IR mbox ,
is normally the file
.B mbox
in your home directory. 
This file can also be changed by setting the
.SB MBOX
environment variable.  Messages remain in the 
.I mbox
file until deliberately removed.
.SH OPTIONS
.LP
If no 
.I recipient 
is specified, 
.B mail
attempts to read messages from the system mailbox.  
.TP 14
.B \-d
Turn on debugging output.  
(Neither particularly interesting nor recommended.)
.TP 
.B \-e
Test for presence of mail.
If there is no mail,
.B mail
prints nothing and exits (with a successful return code).
.TP
.B \-F
Record the message in a file named after the first recipient.
Override the 
.B record 
variable, if set.
.TP
.B \-H
Print header summary only.
.TP
.B \-i
Ignore interrupts (as with the
.B ignore 
variable).
.TP 
.B \-n
Do not initialize from the system default
.B Mail.rc
file.
.TP
.B \-N
Do not print initial header summary.
.TP
.B \-U
Convert
.B uucp
style addresses to Internet standards.
Overrides the 
.B conv
environment variable.
.TP
.B \-v
Pass the
.B \-v
flag to
.BR sendmail (8).
.TP
.BI "\-f\fR [\fP" filename\fR]
Read messages from 
.I filename
instead of system mailbox.  If no 
.I filename
is specified, the 
.I mbox
is used.
.TP
.BI "\-f\fR \fP+" folder
Use the file
.I folder
in the folder directory (same as the
.BR fold er 
command).  The name of this directory is listed in the
.B folder
variable.
.TP
.BI \-h " number"
The number of network \*(lqhops\*(rq made so far.
This is provided for network
software to avoid infinite delivery loops.
.TP
.BI \-r " address"
Pass
.I address
to network delivery software.
All tilde
.RB ( \(ap )
commands are disabled.
.TP
.BI \-s " subject"
Set the
.B Subject
header field to
.IR subject .
.TP
.BI \-T " file"
Print the contents of the
.I article-id
fields of all messages that were read or deleted on
.IR file 
(for the use of
network news programs if available).
.TP
.BI \-u " user"
Read
.IR user 's
system mailbox.
This is only effective if
.IR user 's
system mailbox is not read protected.
.SH USAGE
Refer to 
.TX GSBG
for tutorial information about 
.BR mail .
.SS Starting Mail
.LP
As it starts,
.B mail
reads commands from a system-wide file
.RB ( /usr/lib/Mail.rc \|)
to initialize certain variables,
then it reads from a private start-up file called the
.I \&.mailrc
file (it is normally the file
.B \&.mailrc
in your home directory, but can be changed by setting the
.SB MAILRC
environment variable) for your personal commands and variable settings.  
Most 
.B mail
commands are legal inside start-up files.  The most common uses
for this file are to set up initial display options and alias lists.
The following commands are 
.I not 
legal in the start-up file:
.BR ! ,
.BR C opy,
.BR e dit,
.BR fo llowup,
.BR F ollowup,
.BR ho ld,
.BR m ail,
.BR pre serve,
.BR r eply,
.BR R eply,
.BR replya ll,
.BR replys ender,
.BR sh ell,
and
.BR v isual.
Any errors in the start-up file cause the remaining lines in that file 
to be ignored.
.LP
You can use the 
.B mail
command to send a message directly by including names of recipients as 
arguments on the command line.  When no recipients appear on the 
.B mail
command line, it enters command mode, from which you can read
messages sent to you.  If you list no recipients and have no messages,
.B mail
prints the message:
.RB ` "No mail for \fIusername\fP" '
and exits.
.LP
When in command mode (while reading messages), you can send messages 
using the 
.BR m ail
command.
.SS Sending Mail
.LP
While you are composing a message to send,
.B mail
is in 
.I input 
mode. 
If no subject is specified as an argument to the command
a prompt for the subject is printed. 
After entering the subject line,
.B mail
enters 
.I input 
mode to accept the text of your message to send.
.LP
As you type in the message,
.B mail 
stores it in a temporary file.
To review or modify the message, enter the appropriate 
.IR "tilde escapes" ,
listed below, at the beginning of an input line.
.LP
To indicate that the message is ready to send, type a dot
(or
.SM EOF
character, normally
.SM CTRL-D\s0 )
on a line by itself.  
.B mail
submits the message to
.BR sendmail (8)
for routing to each 
.IR recipient .
.LP
Recipients can be;
.IP \(bu
local usernames
.IP \(bu
Internet addresses of the form:
.RS
.IP
.IB name @ domain
.RE
.IP \(bu
.BR uucp (1C)
addresses of the form:
.RS
.IP
[\fIhost\fB!\fR.\|.\|.\fIhost\fB!\fR]\fIhost\fB!\fIusername
.RE
.IP \(bu
filenames for which you have write permission
.IP \(bu
alias groups
.LP
If the name of the
.I recipient 
begins with a pipe symbol 
.RB ( \||\| ),
the remainder of the name is taken as a shell command to pipe
the message through.  This provides an automatic interface with
any program that reads the standard input, such as
.BR lpr (1)
to record outgoing mail on paper.
An alias group is the name of a list of recipients that is set by the 
.B alias
command, taken from the host's
.B /etc/aliases
file, or taken from the 
Network Information Service
(\s-1NIS\s0)
aliases domain.
See
.BR aliases (5)
for more information about mail addresses and aliases.
.SS Tilde Escapes
.IX  "mail tilde escapes"  ""  "\fLmail\fP tilde escapes"  ""  PAGE START
.IX  "tilde escapes in mail"  ""  "tilde escapes in \fLmail\fP"  ""  PAGE START
The following 
.I tilde escape
commands can be used when composing messages to send. 
Each must appear at the beginning of an input line.
The escape character (\|\(ap\|), can be changed by setting a new
value for the 
.B escape
variable.
The escape character can be entered as text by typing it twice.
.HP
.B \(ap! 
.RI [ shell-command ]
.br
.IX  "mail tilde escapes"  "~!"  "\fLmail\fP tilde escapes"  "\fL\(ap!\fP\ \ \ "
.IX  "~! mail tilde escape"  ""  "\fL\(ap!\fP\ \ \ \(em mail tilde escape"
Escape to the shell.
If present, run
.IR shell-command .
.TP
.B \(ap.
.IX  "mail tilde escapes"  "~."  "\fLmail\fP tilde escapes"  "\fL\(ap.\fP\ \ \ "
.IX  "~. mail tilde escape"  ""  "\fL\(ap.\fP\ \ \ \(em mail tilde escape"
Simulate
.SM EOF
(terminate message input).
.HP
.B \(ap: 
.I mail-command
.br
.PD 0
.HP
.B \(ap_ 
.I mail-command
.br
.IX  "mail tilde escapes"  "~:"  "\fLmail\fP tilde escapes"  "\fL\(ap:\fP\ \ \ "
.IX  "~: mail tilde escape"  ""  "\fL\(ap:\fP\ \ \ \(em mail tilde escape"
.IX  "mail tilde escapes"  "~_"  "\fLmail\fP tilde escapes"  "\fL\(ap_\fP\ \ \ "
.IX  "~_ mail tilde escape"  ""  "\fL\(ap_\fP\ \ \ \(em mail tilde escape"
Perform the indicated
.B mail
command.
Valid only when sending a message while reading mail.
.PD
.TP
.B \(ap?
.IX  "mail tilde escapes"  "~?"  "\fLmail\fP tilde escapes"  "\fL\(ap?\fP\ \ \ "
.IX  "~? mail tilde escape"  ""  "\fL\(ap?\fP\ \ \ \(em mail tilde escape"
Print a summary of tilde escapes.
.TP
.B \(apA
.IX  "mail tilde escapes"  "~A"  "\fLmail\fP tilde escapes"  "\fL\(apA\fP\ \ \ "
.IX  "A"  ""  "\fL\(apA\fP\ \ \ \(em mail tilde escape"
Insert the autograph string 
.B Sign
into the message.
.TP
.B \(apa
.IX  "mail tilde escapes"  "~a"  "\fLmail\fP tilde escapes"  "\fL\(apa\fP\ \ \ "
.IX  "a"  ""  "\fL\(apa\fP\ \ \ \(em mail tilde escape"
Insert the autograph string 
.B sign
into the message.
.TP
\fB\(apb\fI name\fR .\|.\|.
.IX  "mail tilde escapes"  "~b"  "\fLmail\fP tilde escapes"  "\fL\(apb\fP\ \ \ "
.IX  "b"  ""  "\fL\(apb\fP\ \ \ \(em mail tilde escape"
Add the 
.IR name s
to the blind carbon copy
.RB ( Bcc )
list.  This is like the carbon copy
.RB ( Cc )
list, except that the names in the
.B Bcc
list are not shown in the header
of the mail message.
.TP
\fB\(apc\fI name\fR .\|.\|.
.IX  "mail tilde escapes"  "~c"  "\fLmail\fP tilde escapes"  "\fL\(apc\fP\ \ \ "
.IX  "c"  ""  "\fL\(apc\fP\ \ \ \(em mail tilde escape"
Add the 
.IR name s
to the carbon copy
.RB ( Cc )
list.
.TP
.B \(apd
.IX  "mail tilde escapes"  "~d"  "\fLmail\fP tilde escapes"  "\fL\(apd\fP\ \ \ "
.IX  "d"  ""  "\fL\(apd\fP\ \ \ \(em mail tilde escape"
Read in the 
.I dead.letter
file.
The name of this file is listed in the variable
.BR \s-1DEAD\s0 .
.TP
.B \(ape
.IX  "mail tilde escapes"  "~e"  "\fLmail\fP tilde escapes"  "\fL\(ape\fP\ \ \ "
.IX  "e"  ""  "\fL\(ape\fP\ \ \ \(em mail tilde escape"
Invoke the editor to edit the message. 
The name of the editor is listed in the
.SB EDITOR
variable.
The default editor is
.BR ex (1).
.HP
.BI \(apf 
.RI [ message-list ]
.br
.IX  "mail tilde escapes"  "~f"  "\fLmail\fP tilde escapes"  "\fL\(apf\fP\ \ \ "
.IX  "f"  ""  "\fL\(apf\fP\ \ \ \(em mail tilde escape"
Forward the listed messages, or the current message being read.
Valid only when sending a message while reading mail;
the messages are inserted without alteration (as opposed to the
.B \(apm
escape).  
.TP
.B \(aph
.IX  "mail tilde escapes"  "~h"  "\fLmail\fP tilde escapes"  "\fL\(aph\fP\ \ \ "
.IX  "h"  ""  "\fL\(aph\fP\ \ \ \(em mail tilde escape"
Prompt for the message header lines:
.BR Subject ,
.BR To ,
.BR Cc ,
and
.BR Bcc .
If the header line contains text, you can edit the text by
backspacing over it and retyping.
.TP
.BI \(api " variable"
.IX  "mail tilde escapes"  "~i"  "\fLmail\fP tilde escapes"  "\fL\(api\fP\ \ \ "
.IX  "i"  ""  "\fL\(api\fP\ \ \ \(em mail tilde escape"
Insert the value of the named
.I variable
into the message.
.HP
.BI \(apm 
.RI [ message-list ]
.br
.IX  "mail tilde escapes"  "~m"  "\fLmail\fP tilde escapes"  "\fL\(apm\fP\ \ \ "
.IX  "m"  ""  "\fL\(apm\fP\ \ \ \(em mail tilde escape"
Insert text from the specified messages, or the current message,
into the letter.  Valid only when sending a message while reading mail;
the text the message is shifted to the right, and the string contained
in the
.B indentprefix
variable is inserted as the leftmost characters of each line.
If 
.B indentprefix
is not set, a
.SM TAB
character is inserted into each line.
.TP
.B \(app
.IX  "mail tilde escapes"  "~p"  "\fLmail\fP tilde escapes"  "\fL\(app\fP\ \ \ "
.IX  "p"  ""  "\fL\(app\fP\ \ \ \(em mail tilde escape"
Print the message being entered.
.TP
.B \(apq
.IX  "mail tilde escapes"  "~q"  "\fLmail\fP tilde escapes"  "\fL\(apq\fP\ \ \ "
.IX  "q"  ""  "\fL\(apq\fP\ \ \ \(em mail tilde escape"
Quit from input mode by simulating an interrupt.  If the body of the 
message is not empty, the partial message is saved in the 
.I dead.letter
file.
.TP
.BI \(apr " filename"
.PD 0
.TP
.BI \(ap< " filename"
.TP
.BI \(ap<! " shell-command"
.IX  "mail tilde escapes"  "~r"  "\fLmail\fP tilde escapes"  "\fL\(apr\fP\ \ \ "
.IX  "r"  ""  "\fL\(apr\fP\ \ \ \(em mail tilde escape"
.IX  "mail tilde escapes"  "~<"  "\fLmail\fP tilde escapes"  "\fL\(ap<\fP\ \ \ "
.IX  "~< mail tilde escape"  ""  "\fL\(ap<\fP\ \ \ \(em mail tilde escape"
Read in text from the specified file or the standard output of
the specified
.IR shell-command .
.PD
.TP
.BI \(aps " subject"
.IX  "mail tilde escapes"  "~s"  "\fLmail\fP tilde escapes"  "\fL\(aps\fP\ \ \ "
.IX  "s"  ""  "\fL\(aps\fP\ \ \ \(em mail tilde escape"
Set the subject line to
.IR subject .
.TP
.BI \(apt " name .\|.\|."
.IX  "mail tilde escapes"  "~t"  "\fLmail\fP tilde escapes"  "\fL\(apt\fP\ \ \ "
.IX  "t"  ""  "\fL\(apt\fP\ \ \ \(em mail tilde escape"
Add each 
.I name
to the list of recipients.
.TP
.B \(apv
.IX  "mail tilde escapes"  "~v"  "\fLmail\fP tilde escapes"  "\fL\(apv\fP\ \ \ "
.IX  "v"  ""  "\fL\(apv\fP\ \ \ \(em mail tilde escape"
Invoke a visual editor to edit the message.  The name of the editor
is listed in the
.SB VISUAL
variable.  The default visual editor is
.BR vi (1).
.TP
.BI \(apw " filename"
.IX  "mail tilde escapes"  "~w"  "\fLmail\fP tilde escapes"  "\fL\(apw\fP\ \ \ "
.IX  "w mail tilde escape"  ""  "\fL\(apw\fP\ \ \ \(em mail tilde escape"
Write the message text onto the given file, without the header.
.TP
.B \(apx
.IX  "mail tilde escapes"  "~x"  "\fLmail\fP tilde escapes"  "\fL\(apx\fP\ \ \ "
.IX  "x mail tilde escape"  ""  "\fL\(apx\fP\ \ \ \(em mail tilde escape"
Exit as with
.B \(apq
but do not save the message in the
.I dead.letter
file.
.TP
.BI \(ap| " shell-command"
.IX  "mail tilde escapes"  "~|"  "\fLmail\fP tilde escapes"  "\fL\(ap|\fP\ \ \ "
.IX  "~| mail tilde escape"  ""  "\fL\(ap|\fP\ \ \ \(em mail tilde escape"
Pipe the body of the message through the given 
.IR shell-command .
If 
.I shell-command
returns a successful exit status,
the output of the command replaces the message.
.IX  "mail tilde escapes"  ""  "\fLmail\fP tilde escapes"  ""  PAGE END
.IX  "tilde escapes in mail"  ""  "tilde escapes in \fLmail\fP"  ""  PAGE END
.IX  "tilde escapes in mail"  ""  "tilde escapes in \fLmail\fP"  ""  PRINT "see also \fLmail\fP tilde escapes"
.SS Reading Mail
When you enter
.I command 
mode in order to read your messages, 
.B mail
displays a header summary of the first several messages,
followed by a prompt for one of the commands listed below.
The default prompt is the
.B &
(ampersand character).
.LP
Message are listed and referred to by number.
There is, at any time, a 
.B current
message, which is marked by a
.B >
in the header summary.  
For commands that take an optional list of messages,
if you omit a message number as an argument, the command applies
to the current message.
.LP
A 
.I message-list
is a list of message specifications, separated by
.SM SPACE
characters, which may include:
.RS
.TP
.B \&.
The current message.
.PD 0
.TP
.I n
Message number
.IR n .
.TP
.B ^
The first undeleted message.
.TP
.B $
The last message.
.TP
.B +
The next undeleted message.
.TP
.B \-
The previous undeleted message.
.TP
.B *
All messages.
.TP
.IB n \- m
An inclusive range of message numbers.
.TP
.I user
All messages from
.IR user .
.TP
.BI / string
All messages with
.I string
in the subject line (case ignored).
.TP
.BI : c
All messages of type
.IR c ,
where
.I c
is one of:
.sp .5
.RS
.RS
.TP
.B d
deleted messages
.TP
.B n
new messages
.TP
.B o
old messages
.TP
.B r
read messages
.TP
.B u
unread messages
.RE
.RE
.IP
Note: the context of the command determines whether this type of
message specification makes sense.
.PD
.RE
.LP
Additional arguments are treated as strings whose usage
depends on the command involved.  Filenames, where expected,
are expanded using the normal shell filename-substitution
mechanism.
.LP
Special characters, recognized by certain commands, are
documented with those commands.
.SS Commands
.IX  "mail commands"  ""  "\fLmail\fP commands"  ""  PAGE START
While in command mode, if you type in an empty command line
(a
.SM RETURN
or
.SM NEWLINE
only), the
.BR p rint
command is assumed.  The following is a complete list of 
.B mail
commands:
.TP 18
.BI ! " shell-command"
.IX  "mail commands"  "!"  "\fLmail\fP commands"  "\fL!\fP"
.IX  "! mail command"  ""  "\fL!\fP mail command"
Escape to the shell.  The name of the shell to use is listed in
the 
.SB SHELL
variable.
.TP
.BI # " arguments"
.IX  "mail commands"  "#"  "\fLmail\fP commands"  "\fL#\fP"
.IX  "# mail command"  ""  "\fL#\fP mail command"
Null command.  This may be used as if it were a comment in 
.I \&.mailrc
files, but note that it must be separated from its arguments
(commentary) by white space.
.TP
.B =
.IX  "mail commands"  "="  "\fLmail\fP commands"  "\fL=\fP"
.IX  "= mail command"  ""  "\fL=\fP mail command"
Print the current message number.
.TP
.B ?
.IX  "mail commands"  "?"  "\fLmail\fP commands"  "\fL?\fP"
.IX  "? mail command"  ""  "\fL?\fP mail command"
Print a summary of commands.
.HP
.BR a lias 
.RI [ "alias recipient" \|.\|.\|.\^]
.br
.PD 0
.HP
.BR g roup 
.RI [ "alias recipient" \|.\|.\|.\^]
.br
.IX  "mail commands"  "alias"  "\fLmail\fP commands"  "\fLalias\fP"
.IX  "alias mail command"  ""  "\fLalias\fP mail command"
.IX  "mail commands"  "group"  "\fLmail\fP commands"  "\fLgroup\fP"
.IX  "group mail command"  ""  "\fLgroup\fP mail command"
Declare an alias for the given list of recipients. 
The list will be substituted when the
.I alias
is used as a recipient while sending mail.  When put in the
.I \&.mailrc
file, this command provides you with a record of the alias.  With no 
arguments, the command displays the list of defined aliases.
.PD
.TP
.BR alt "ernates\fI name \fP.\|.\|."
.IX  "mail commands"  "alternates"  "\fLmail\fP commands"  "\fLalternates\fP"
.IX  "alternates mail command"  ""  "\fLalternates\fP mail command"
Declare a list of alternate names for your login.
When responding to a message,
these names are removed from the list of recipients for the response.
With no arguments,
.BR alt ernates
prints the current list of alternate names.
.HP
.BR cd [ 
.IR directory ]
.br
.PD 0
.TP
.BR ch "dir [\fIdirectory\fP]"
.IX  "mail commands"  "cd"  "\fLmail\fP commands"  "\fLcd\fP"
.IX  "cd mail command"  ""  "\fLcd\fP mail command"

usr.bin/sunwindows/sunview/defaults/lib/Mail.d
SunDefaults_Version 2
;	@(#)Mail.d 1.1 92/07/30 SMI
/Mail 			""
//$Specialformat_to_defaults "mailrc_to_defaults"
//$Defaults_to_specialformat "defaults_to_mailrc"

//Set/Mess1/$Message	"    ~~~~~~ Mailtool related options ~~~~~~"

//Set/allowreversescan	"No"
	$Enumeration	""
	$Help		"Enables/disables processing messages in reverse direction."
	No		"No"
	No/$Help	"Messages will be processed in the forward direction, i.e., first in, first processed."
	Yes		"Yes"
	Yes/$Help	"Messages will be processed in the reverse direction, i.e., last in, first processed."

//Set/alwaysusepopup	"No"
	$Enumeration	""
	$Help		"Controls creation of message composition window."
	No		"No"
	No/$Help	"Create message composition window by splitting message viewing window."
	Yes		"Yes"
	Yes/$Help	"Create message composition window by popping up a separate frame."
	
//Set/askbcc		"No"
	$Enumeration	""
	$Help		"Enables/disables prompting user for 'bcc' field when sending."
	No		"No"
	No/$Help	"Do NOT automatically prompt user for 'bcc' field when sending."
	Yes		"Yes"
	Yes/$Help	"Automatically prompt user for 'bcc' field when sending."

//Set/bell		"0"
	$Help		"Number of times to ring the audible bell when new mail arrives."

//Set/disablefields	"No"
	$Enumeration	""
	$Help		"Enables/disables use of text fields in message composition window."
	No		"No"
	No/$Help	"Use fields. Allows user to advance to next (previous) field via CTRL-TAB (SHIFT-CTRL-TAB)."
	Yes		"Yes"
	Yes/$Help	"Do not use fields."

//Set/editmessagewindow	"Yes"
	$Enumeration	""
	$Help		"Enables/disables editing of messages in message subwindow."
	No	"No"
	No/$Help	"The first edit in the message subwindow will ask for confirmation."
	Yes		"Yes"
	Yes/$Help	"Allows editing of message subwindow. Confirmation requested when edits are saved."
	$Always_Save	"Yes"
	
//Set/expert		"No"
	$Enumeration	""
	$Help		"Enables/disables expert mode, i.e., no confirmations."
	No		"No"
	No/$Help	"Disable expert mode. Request confirmations."
	Yes		"Yes"
	Yes/$Help	"Operate in expert mode. Do not request confirmations."
	
//Set/filemenu
	$Help		"List of files to initialize the 'File:' menu, e.g,. \"+mbox +trash\"."

//Set/filemenusize	"10"
	$Help		"Maximum number of files cached in the File: menu."

//Set/flash		"0"
	$Help		"Number of times to flash the Mailtool window or icon when new mail arrives."

//Set/interval		"300"
	$Help		"Number of seconds to wait before checking again for new mail."

//Set/moveinputfocus	"No"
	$Enumeration	""
	$Help		"Specifies whether or not to move input focus to composition window when it is created."
	No		"No"
	No/$Help	"Only move input focus if it is already in one of the mailtool's subwindows."
	Yes		"Yes"
	Yes/$Help	"Always move input focus. Enables user to immediately start typing response."

//Set/msgpercent	"50"
	$Help		"Percent of message window to use for displaying current message while composing a new one."

//Set/printmail
	$Help		"Commandline to be used for printing, e.g., 'pr -h mail | lpr -Pimagen'." Default is "lpr -p".

//Set/trash
	$Help		"Name of trash folder. Deleted messages stay here until you hit 'Done'."


//Set/Mess2/$Message	""
//Set/Mess3/$Message	"    ~~~~~~ options relating to appearance of mailtool ~~~~~~"
//Set/headerlines	"10"
	$Help		"Number of lines in header subwindow."

//Set/maillines		"30"
	$Help		"Number of lines in message subwindow."

//Set/popuplines		"30"
	$Help		"Number of lines in popup message composition subwindow."

//Set/Mess6/$Message	""
//Set/Mess7/$Message	"    ~~~~~~ options affecting both Mailtool and Mail ~~~~~~"
	
//Set/allnet		"No"
	$Enumeration	""
	$Help		"Affects how comparisons of recipient addresses are performed."
	No		"No"
	No/$Help	"Comparisons of recipient addresses use the entire recipient string."
	Yes		"Yes"
	Yes/$Help	"Comparisons of recipient addresses use only the base user name."

//Set/alwaysignore	"No"
	$Enumeration	""
	$Help		"Determines which commands are affected by 'ignore' settings."
	No		"No"
	No/$Help	"Only the 'print' command is affected by 'ignore' settings."
	Yes		"Yes"
	Yes/$Help	"All commands are affected by the 'ignore' settings, e.g., save, copy, etc."

//Set/append		"Yes"
	$Enumeration	""
	$Help		"Specifies whether messages are appended or prepended to 'mbox'."
	No		"No"
	No/$Help	"Prepend messages at beginning of 'mbox'."
	Yes		"Yes"
	Yes/$Help	"Append messages to end of 'mbox'."
	Yes		"Yes"

//Set/ask		"No"
	$Enumeration	""
	$Help		"No longer implemented. Use 'asksub'."
	Yes		"Yes"
	No		"No"

//Set/askcc		"No"
	$Enumeration	""
	$Help		"Enables/disables prompting user for 'cc' field when sending."
	No		"No"
	No/$Help	"Do NOT automatically prompt user for 'cc' field when sending."
	Yes		"Yes"
	Yes/$Help	"Automatically prompt user for 'cc' field when sending."

//Set/asksub		"Yes"
	$Enumeration	""
	$Help		"Enables/disables prompting user for 'subject' field when sending."
	No		"No"
	No/$Help	"Do NOT automatically prompt user for 'subject' field when sending."
	Yes		"Yes"
	Yes/$Help	"Automatically prompt user for 'subject' field when sending."

//Set/autoprint		"No"
	$Enumeration	""
	$Help		"Enables/disables displaying the next message when current message is deleted."
	No		"No"
	No/$Help	"Do NOT automatically display next message when current message is deleted."
	Yes		"Yes"
	Yes/$Help	"Automatically display next message when current message is deleted."

//Set/DEAD		"~/dead.letter"
	$Help		"Full path name of file where partial messages are stored in case of interrupt or delivery error."

//Set/folder
	$Help		"Directory in which to store mail folders."

//Set/hold		"Yes"
	$Enumeration	""
	$Help		"Enables/disables preserving messages that are read in the mailbox."
	No		"No"
	No/$Help	"Move messages that are read into 'mbox'."
	Yes		"Yes"
	Yes/$Help	"Hold messages in mailbox."
	$Always_Save	"Yes"

//Set/indentprefix	"	"
	$Help		"Inserted in front of message when message is indented. Defaults to tab."

//Set/keep		"No"
	$Enumeration	""
	$Help		"Enables/disables deleting system mailbox when empty."
	No		"No"
	No/$Help	"When mailbox is empty, delete it."
	Yes		"Yes""
	Yes/$Help	"When mailbox is empty, truncate it to zero length."

//Set/keepsave		"No"
	$Enumeration	""
	$Help		"Enables/disables deleting messages that are saved in folders."
	No		"No"
	No/$Help	"Delete messages that are saved in folders."
	Yes		"Yes"
	Yes/$Help	"Do NOT delete messages that are saved in folders."

//Set/LISTER		"ls -F"
	$Help		"Command (and options) for listing folders. 'ls -F' shows subdirectories as pullrights."

//Set/MBOX		"~/mbox"
	$Help		"Full path name of file in which to save messages that have been read. See 'hold' option."

//Set/metoo		"No"
	$Enumeration	""
	$Help		"Enables/disables including the sender if listed in an alias."
	No		"No"
	No/$Help	"Do NOT include sender when listed in an alias."
	Yes		"Yes"
	Yes/$Help	"Include sender when listed in an alias."

//Set/onehop		"No"
	$Help		"Enables/disables alteration of addresses when Replying, enabling direct response where possible."
	$Enumeration	""
	No		"No"
	No/$Help	"Recipients' addresses are altered to be relative to originating author's machine."
	Yes		"Yes"
	Yes/$Help	"Recipients' addresses are not altered."

//Set/outfolder		"No"
	$Enumeration	""
	$Help		"Files to record outgoing mail are located in the 'folder' directory. See 'record' option."
	No		"No"
	No/$Help	"Do NOT save outgoing mail.  See 'Set/record' below."
	Yes		"Yes"
	Yes/$Help	"Save outgoing mail.  See 'Set/record' below."
//Set/record
	$Help		"File in which to save all outgoing mail."
//Set/replyall		"No"
	$Enumeration	""
	$Help		"Reverse action of reply and Reply."
	No		"No"
	No/$Help	"'r' means reply to just to sender, 'R' means reply to all recipients."
	Yes		"Yes"
	Yes/$Help	"'r' means reply to all recipients, 'R' means reply to just to sender."

//Set/save		"Yes"
	$Enumeration	""
	$Help		"Enables/disables saving partial messages in the file specified by the value of DEAD."
	No		"No"
	No/$Help	"Do NOT save partial messages in file specified by the value of DEAD."
	Yes		"Yes"
	Yes/$Help	"Save partial messages in file specified by the value of DEAD."

//Set/sendmail		"sendmail"
	$Help		"Alternate command for delivering messages."

//Set/showto		"No"
	$Enumeration	""
	$Help		"Enables/disables showing name of recipient instead of author when message is from you."
	No		"No"
	No/$Help	"Always show sender in the header line"
	Yes		"Yes"
	Yes/$Help	"If message was sent by the reader, show recipient instead of sender in the header line"

//Set/Mess8/$Message	""
//Set/Mess9/$Message	"    ~~~~~~ options that affect only the program 'Mail'  ~~~~~~"
	
//Set/bang		"No"
	$Enumeration	""
	$Help		"Enable/disable special-casing of exclamation points (!) in shell escape commands."
	No		"No"
	No/$Help	"Enable special-casing of exclamation points (!) in shell escape commands."
	Yes		"Yes"
	Yes/$Help	"Disable special-casing of exclamation points (!) in shell escape commands."

//Set/cmd
	$Help		"Default command for the pipe command. (Wizards option.)"

//Set/conv
	$Help		"Address style for conversion of uucp addresses."

//Set/crt
	$Help		"Pipe messages having more than this number of lines through command specified by 'Pager'."

//Set/dot		"Yes"
	$Enumeration	""
	$Help		"Accept '.' alone on line to terminate message."
	No		"No"
	Yes		"Yes"

//Set/EDITOR		"ex"
	$Help		"The command to run when the edit or '~e' command is used."

//Set/escape
	$Help		"Escape character to be used instead of ~."

//Set/header		"Yes"
	$Enumeration	""
	$Help		"Enable/disable printing of the header summary when entering mail."
	No		"No"
	No/$Help	"Do not print header summary when entering mail."
	Yes		"Yes"
	Yes/$Help	"Print header summary when entering mail."

//Set/ignore		"No"
	$Enumeration	""
	$Help		"Enable/disable ignoring interrupts while entering messages. Handy for noisy dialup lines."
	No		"No"
	No/$Help	"Notice ^C's (RUBOUT) while sending mail."
	Yes		"Yes"
	Yes/$Help	"Ignore ^C's (RUBOUT) while sending mail."

//Set/ignoreeof		"No"
	$Enumeration	""
	$Help		"Enable/disable terminating messages/command inputs with ^D."
	No		"No"
	No/$Help	"Terminate letters/command input with ^D. "
	Yes		"Yes"
	Yes/$Help	"Input must be terminated by a period on a line by itself, or by the '~.' command."


//Set/page		"No"		
	$Enumeration	""
	$Help		"Enables/disables inserting form feed after each message sent through the pipe."
	No		"No"
	No/$Help	"Do NOT insert form feed after each message."
	Yes		"Yes"
	Yes/$Help	"Insert form feed after each message."

//Set/PAGER		"more"
	$Help		"Command to be used as a filter for paginating output. See 'crt' option."

//Set/prompt		"& "
	$Help		"Command mode prompt."

//Set/quiet		"No"
	$Enumeration	""
	$Help		"Enables/disables suppressing printing of opening message and version number."
	No		"No"
	No/$Help	"Print opening message and version number when entering Mail."
	Yes		"Yes"
	Yes/$Help	"Suppress printing of opening message and Mail version."

//Set/screen
	$Help		"Sets the number of lines in a screen-full of headers for the headers command."

//Set/sendwait		"No"
	$Enumeration	""
	$Help		"Enables/disables waiting for background mailer to finish before returning."
	No		"No"
	No/$Help	"Do not wait for background mailer to finish before returning."
	Yes		"Yes"
	Yes/$Help	"Wait for background mailer to finish before returning."

//Set/SHELL		"sh"
	$Help		"Name of the command interpreter to be used in the '!' command and '~!' escape."

//Set/sign
	$Help		"Variable inserted into the text of the message when '~a' (autograph) command is given."

//Set/toplines		"5"
	$Help		"Number of lines of header to be printed by the 'top' command."
	
//Set/verbose		"No"
	$Enumeration	""
	$Help		"Enables/disables invoking 'sendmail' with the '-v'."
	No		"No"
	No/$Help	"Do NOT 'sendmail' with the '-v' flag."
	Yes		"Yes"
	Yes/$Help	"Invoke 'sendmail' with the '-v' flag."

//Set/VISUAL		"vi"
	$Help		"Name of screen editor to be used with visual or '~v' command."




